test/bdd/steps/hooks.js

```diff
@@ -29,6 +29,7 @@ Before(function (testCase, done) {
     this.state.localBlockchain = null;
     this.state.nodes = [];
     this.state.bootstraps = [];
+    this.state.manualStuff = {};
```

Comment:
in case there will be following commits,  I would appreciate more clear and meaningful property name here

Reply:
Of course.